tests/js/spec/views/twoFactorRequired.spec.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {Client} from 'app/api';
+import TwoFactorRequired from 'app/views/settings/account/accountSecurity/';
+
+const ENDPOINT = '/users/me/authenticators/';
+const ORG_ENDPOINT = '/organizations/';
+
+describe('TwoFactorRequired', function() {
+  beforeEach(function() {
+    Client.clearMockResponses();
+  });
+
+  it('renders empty', function() {
+    Client.addMockResponse({
```

Comment:
I would move whichever response is used the most into a `beforeEach` and just override the mock response in a test if it differs.